src/components/molecules/AdminNavBar/components/SidebarContent/SidebarContent.tsx

```diff
@@ -0,0 +1,128 @@
+import { useMemo } from "react";
+import { Link, useLocation } from "react-router-dom";
+import {
+  CalendarIcon,
+  ChartBarIcon,
+  CogIcon,
+  MapIcon,
+  SwitchHorizontalIcon,
+  UsersIcon,
+} from "@heroicons/react/outline";
```

Comment:
I think this should be replaced with FA ones. If it can be done in this PR - great. If too much work, we'll need a `@debt` comment since I think we'll be using a single icon library

Reply:
using FA icons: https://github.com/sparkletown/sparkle/pull/2812/commits/17b677903b65f4efb0ea32fd3b8b5b53b2094545

some of the needed icons are still in beta version so they're not available in the code, replaced with more or less similar ones.